docs/explore/datadog/index-datadog.mdx

```diff
@@ -0,0 +1,484 @@
+
+##### Characteristics
+
+
+Since Redis is popular due to performance, generally you would expect most operations to return in single digit milliseconds. Tune any alerts to match your SLA. It’s generally recommended that you also measure Redis operation latency at the client side to make it easier to determine if a server slow down or an increase in network latency is the culprit in any performance issues.
+
+##### Possible Causes
+
+|   Cause    | Factors     |
+| ---------- | ----------- | 
+| Spike in requests | Check both the Network Traffic and Operations Per Second metrics to determine if there is a corresponding increase |
+| Slow-running queries | Check the slow log in the Redis Enterprise UI for the database |
+| Insufficient compute resources | Check to see if the CPU Usage, Memory Usage Percentage, or Evictions are increasing |
+
+
+##### Remediation
+
+
+|   Action   | Method     |
+| ---------- | ----------- | 
+| Increase resources | The database can be scaled up online by going to the Web UI and enabling clustering on the database. In extreme cases, more nodes can be added to the cluster and resources rebalanced |
+| Inefficient queries | Redis allows you to view a slow log with a tunable threshold. It can be viewed either in the Redis Enterprise UI or by running:   redis-cli -h HOST -p PORT -a PASSWORD SLOWLOG GET 100 |
+
+
+
+
+### 2. Memory Usage Percentage
+
+#### Definition
+
+##### redis_enterprise.memory_usage_percent (unit: percentage)
+
+This is the percentage of used memory over the memory limit set for the database.
+
+##### Characteristics
+
+
+In Redis Enterprise, all databases have a maximum memory limit set to ensure isolation in a multi-tenant environment. This is also highly recommended when running open source Redis. Be aware that Redis does not immediately free memory upon key deletion. Depending on the size of the database, generally between 80-95% is a safe threshold.
+
+##### Possible Causes
+
+|   Cause    | Factors     |
+| ---------- | ----------- | 
+| Possible spike in activity | Check both the Network Traffic and Operations Per Second metrics to determine if there is a corresponding increase |
+| Database sized incorrectly | View the Memory Usage raw bytes over time to see if a usage pattern has changed |
+| Incorrect retention policies | Check to see if keys are being Evicted or Expired |
+
+
+
+
+##### Remediation
+
+|   Action    | Method     |
+| ---------- | ----------- |
+| Increase resources | The database memory limit can be raised online with no downtime through either the Redis Enterprise UI or the API |
+| Retention Policy | In a caching use case, setting a TTL for unused data to expire is often helpful. In addition, Eviction policies can be set, however, these may often not be able to keep up in extremely high throughput environments with very tight resource constraints |
+
+
+### 3. Cache Hit Rate
+
+##### Definition
+
+#####  redis_enterprise.cache_hit_rate (unit: percent)
```

Comment:
See earlier comments about whether `_` is required here or not, I don't know - just checking for consistency.